config/locales/en.yml

```diff
@@ -24,6 +24,9 @@ en:
               title: General information
             save: Create
             title: New delegation
+            select_member: Select member
+            grantee: Grantee
+            granter: Granter
```

Comment:
I could use the ones in index instead

Reply:
I think the duplication in this case is fine. It gives us flexibility